queries/commits_query.py

```diff
@@ -3,14 +3,20 @@
 from db_manager.AugurInterface import AugurInterface
 
 
-def commits_query(dbmc, repo_ids):
+def commits_query(dbmc: AugurInterface, repo_ids: list(str)) -> dict:
```

Comment:
Why is this one different than the other queries?

Reply:
ah good question, that was the first one I updated, forgot to do this to the other queries as well. these type annotations are "nice to have"s in that they're not enforced strictly like they are in other languages, but I need to also be consistent.

Reply:
@JamesKunstle will mark as resolved once the consistency is there

Reply:
on my side I still see (dbmc: AugurInterface, repo_ids: list(str)) -> dict: for the commits query and (dbmc, repo_ids): for the rest

Reply:
It was changed in the most recent commit to this branch, it's in files changed.

Reply:
checked on it, up to date.